docs/userguide/icu/icu4j-readme/index.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: default
 title: ICU4J Readme
-nav_order: 10
+nav_order: 8
 parent: ICU
+has_children: true
 ---
```

Comment:
```
docs/userguide/icudata.md → docs/userguide/icu/icudata.md
```
I can't figure out how to comment on a file move without contents changes...

Shouldn't this move to icu_data/index.md?

Reply:
Done. Yes, thanks.